docs/source/actions.rst

```diff
@@ -102,6 +102,12 @@ Default: ``None``
 SPIDERMON_BODY_HTML_TEMPLATE
 ----------------------------
 
+String containing the location of the template for the spidermon email report.
```

Comment:
:lipstick: 

```suggestion
String containing the location of the template for the Spidermon email report.
```

Also, I think it would be great to have here a link to a topic that covers how to write such a template. Since at this point there is no such a topic, I think that we could say ‘Jinja2 template’ instead of just ‘template’, and link ‘Jinja2’ to the Jinja2 documentation.